### DISCUSSION TITLE
Uploade doc sur demande de réservation .

### DISCUSSION THREAD

**Discussion by taib27** [JeromeDevome/GRR#318] [General]
Bonjour , 
Est-il possible d'ajouter un champs pour uploader un document (jpeg, PNG) sur la réservation .
je n'ai pas trouvé le moyen de le faire depuis les Champs additionnels.
V4.2.0
Merci par avance

**Comment by JeromeDevome** [comment]
Bonjour, 
Non pour le moment cela reste une suggestion #244

**Reply by taib27** [reply]
merci pour l retour
